responsibleai/environments/responsibleai-ubuntu20.04-py38-cpu/tests/data/training-mltable-folder/training-machine-dataset.csv

```diff
@@ -0,0 +1,190 @@
+VendorName,ModelName,MYCT,MMIN,MMAX,CACH,CHMIN,CHMAX,PRP,ERP
```

Comment:
wonder if this can just be downloaded somehow instead of being added to the repo?

Reply:
Yeah, I tried downloading from blob store but somehow AutoML run is not able to access the dataset from blob link.